core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -632,13 +530,7 @@ public static CLICommand clone(String name) {
 
     private static final Logger LOGGER = Logger.getLogger(CLICommand.class.getName());
 
-    /**
-     * Key for {@link Channel#getProperty(Object)} that links to the {@link Authentication} object
-     * which captures the identity of the client given by the transport layer.
-     */
-    public static final ChannelProperty<Authentication> TRANSPORT_AUTHENTICATION = new ChannelProperty<>(Authentication.class, "transportAuthentication");
```

Comment:
:bug: it would break the Jenkins module: https://github.com/jenkinsci/ssh-cli-auth-module/blob/1dc5c9625ed485123216dc995e159c1383eaddef/src/main/java/org/jenkinsci/main/modules/cli/auth/ssh/SshCliAuthenticator.java#L44

Reply:
No, because `CliTransportAuthenticator` is no longer used.

Reply:
That brings up a good point, though, which is that this module should be patched to rip out the deprecated stuff, and that can be integrated here. I am preparing a patch for that.